PalasoUIWindowsForms.GeckoFxWebBrowserAdapter/WebThumbnailViewer.cs

```diff
@@ -133,7 +133,16 @@ public void LoadItems(IEnumerable<string> pathList)
 				sb.AppendLine("<div class='imageWrap' id='" + htmlPath
 					+ "'><div class='imageWrapRel'><div class='imageWrapOuter'><div class='imageWrapMid'><div class='imageWrapInner'>");
 				// the data-echo attribute is part of lazy loading for very long lists (see below).
-				sb.AppendLine("<img class='image' src='' data-echo='file://" + new Uri(GetThumbnailFile(path)).AbsolutePath + "'>");
+
+				if (PlatformUtilities.Platform.IsWindows)
+				{
+					sb.AppendLine("<img class='image' src='' data-echo='file://" + htmlPath + "'>");
+				}
+				else
+				{
+					sb.AppendLine("<img class='image' src='' data-echo='file://" + new Uri(GetThumbnailFile(path)).AbsolutePath + "'>");
```

Comment:
I'm wondering if we could enhance EnhancedImageServer so that we don't have to render all thumbnails immediately but instead when the browser wants to display one?